_tour/singleton-objects.md

```diff
@@ -49,7 +49,7 @@ The `info` method is visible because of the import statement, `import logging.Lo
 
 Imports require a "stable path" to the imported symbol, and an object is a stable path.
 
-Note: If an `object` is not top-level but is nested in another class or object, then the object is "path-dependent" like any other member. This means that given two kinds of beverages, `class Milk` and `class OrangeJuice`, a class member `object NutritionInfo` "depends" on the enclosing instance, either milk or orange juice. `milk.NutritionInfo` is entirely distinct from `oj.NutritionInfo`.
+Note: If an `object` is not top-level but is nested in another class or object, then the object is "path-dependent" like any other member. This means that given two kinds of beverages, `class Milk` and `class OrangeJuice`, a class member `object NutritionInfo` "depends" on the enclosing instance, either milk or orange juice. `Milk.NutritionInfo` is entirely distinct from `OrangeJuice.NutritionInfo`.
```

Comment:
`Milk` and `OrangeJuice` are not `Object`s but classes. I think the original note is trying to say that there are `milk` and `obj` instances created with `val milk = new Milk` and `val oj = new OrangeJuice`.

But apparently this note is not clear enough so maybe we can find a better wording for it, or we can make a link to https://docs.scala-lang.org/tour/inner-classes.html ?